guides/users/custom-authentication.md

```diff
@@ -0,0 +1,172 @@
+# Custom authentication
+
+*You can use the official [`solidus_auth_devise`][solidus-auth-devise] gem
+to provide a `Spree::User` model and basic authentication for Solidus. See its
+documentation for additional setup instructions.*
+
+Stores require a `User` model in order to take advantage of all of Solidus's
+features. This model can have any name, and Solidus can integrate with your
+existing Rails application's existing `User` model.
+
+By default, Solidus provides a [`Spree::LegacyUser` model][legacy-user] that
+offers the bare minimum functionality of a user. The model is intended to be
+modified by extensions.
+
+The rest of this article outlines the steps required should you decide to create
+a `User` model from scratch, use an authentication solution like
+[Devise][devise], or integrate your application's existing `User` model.
```

Comment:
Missing link

Reply:
Also this article does not handle the usage of Devise. Do you plan to extend this article? If not this paragraph should be removed as this confuses

Reply:
I just ran through the requirements in this article with a `User` model that I set up with Devise. If someone was already using Devise for their user model they could use this article to integrate it with Solidus.

Why do you feel that mentioning Devise here as an example confuses?

Reply:
Ah, ok, I misread the comma and the "or". Makes sense. Sorry for the confusion.